spec/features/parks/01_index_spec.rb

```diff
@@ -9,7 +9,11 @@
 
   scenario "visitor signs in and views parks on index page" do
 
-    parks = FactoryGirl.create_list(:park, 2)
+    parks = FactoryGirl.create_list(:park_with_reviews, 2)
+    # parks.each do |park|
+    #   # create a review for that park
+    # end
+
```

Comment:
Why is this commented code here? Should not be submitting PRs with commented-out code.